featctl/cmd/describe_entity.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var describeEntityCmd = &cobra.Command{
-	Use:   "entity",
+	Use:   "entity [entity_name]",
```

Comment:
Suggestion: Use `<>` if the arg is required

Reply:
done